roles/vault/defaults/main.yml

```diff
@@ -18,12 +18,15 @@ vault_roles_dir: "{{ vault_base_dir }}/roles"
 vault_secrets_dir: "{{ vault_base_dir }}/secrets"
 vault_log_dir: "/var/log/vault"
 
+# Arch of Docker images and needed packages
+image_arch: "{{ image_arch | default('amd64') }}"
```

Comment:
This is not how defaults works. Just put it like this:

# Arch of Docker images and needed packages
image_arch: amd64

There's no need to quote if it starts with a letter.